roles/docker/tasks/main.yml

```diff
@@ -7,6 +7,7 @@
 
 - set_fact:
     l_use_system_container: "{{ openshift.docker.use_system_container | default(False) }}"
+    l_replace_docker_with_crio: "{{ openshift.docker.replace_docker_with_crio | default(False) }}"
```

Comment:
can we call `openshift.docker.replace_docker_with_crio` to something like `.use_crio`?  If we use something like `.use_crio`, it will probably keep the same name in future